corehq/warehouse/transforms/__init__.py

```diff
@@ -0,0 +1,17 @@
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+)
+
+
+def app_status_transform(row):
+    print row
```

Comment:
if we really need to do python transforms i think we could select from the staging table, perform the transformations in python, and then do a bulk insert into an intermediate table, and then update the final table from there. we basically want to avoid updating one record at a time